cpp/ql/test/query-tests/Likely Bugs/Likely Typos/FutileParams/test.c

```diff
@@ -1,18 +1,29 @@
-void foo();
-void bar(void);
-void baz(int);
+void declared_empty();
+void declared_void(void);
+void declared_with(int);
+void declared_empty_defined_with();
 
 void test() {
-  foo(); // GOOD
-  foo(1); // BAD
-  bar(); // GOOD
-  baz(1); // GOOD
+  declared_empty(); // GOOD
+  declared_empty(1); // BAD
+  declared_void(); // GOOD
+  declared_with(1); // GOOD
   
   undeclared(1); // GOOD
   
   not_yet_declared1(1); // BAD
   not_yet_declared2(1); // GOOD
+
+  declared_empty_defined_with(); // BAD
+  declared_empty_defined_with(1); // GOOD
+
+  int x;
+  declared_empty_defined_with(&x); // BAD
+  declared_empty_defined_with(x, x); // BAD
```

Comment:
Please mark both of these as `// BAD [NOT DETECTED]`

Reply:
Done; I think these (and the above instance) are best covered by a separate query.